Pode.sln

```diff
@@ -0,0 +1,30 @@
+﻿
```

Comment:
I saw this was accidentally added before a couple days back, and then removed. Just needs removing again :)

Reply:
Visual Studio Code automatically create the sln file 
I just disabled the feature